lib/Folder/FolderManager.php

```diff
@@ -59,10 +69,14 @@ public function getAllFolders(): array {
 		$folderMap = [];
 		foreach ($rows as $row) {
 			$id = (int)$row['folder_id'];
-			$folderMap[$id] = [
+			$applicables = $applicableMap[$id] ?? [];
+			if (empty($applicables)) {
+				$applicables = new stdClass();
```

Comment:
This is a controller specific thing that ends up in the manager, is there no way to avoid that?